tests/crypto/enclave/host/host.c

```diff
@@ -1,8 +1,13 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#if defined(_WIN32)
+#pragma warning(disable : 4996)
```

Comment:
Can this be moved down to whatever it's trying to wrap, surrounded by a push/pop?